src/Http/Integration/Strava/Requests/AthleteActivityRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Integrations\Strava\Requests;
+
+use Saloon\Enums\Method;
+use Saloon\Http\Request;
+
+class AthleteActivityRequest extends Request
+{
+    /**
+     * The HTTP method of the request
+     */
+    protected Method $method = Method::GET;
+
+    public function __construct(
+        private array $payload,
+    ) {}
+
+    /**
+     * The endpoint for the request
+     */
+    public function resolveEndpoint(): string
+    {
+        return '/athlete/activities?page='.$this->payload['page'].'&per_page='.$this->payload['per_page'];
+    }
```

Comment:
probably wanna do a nicer query string builder here or something its kinda blaggh hard to see what all is being used here